README.md

```diff
@@ -197,6 +214,48 @@ Delete the oldest 3 version of a package hosted in a different repo than the one
     token: ${{ secrets.GITHUB_PAT }}
 ```
 
+<br>
+
+### Delete oldest x number of versions and keeping y latest versions of a package hosted in the same repo as the workflow
+
+To delete oldest x number of versions while keeping minimum y latest versions of a package hosted in the same repo as the workflow the __package-name__, __num-old-versions-to-delete__, and __min-versions-to-keep__ inputs are required.
+
+__Example__
+
+Delete the oldest 3 version and always keep atleast 2 versions of a package hosted in the same repo as the workflow
+
+```yaml
+- uses: actions/delete-package-versions@v1
+  with:
+    package-name: 'test-package'
+    num-old-versions-to-delete: 3
+    min-versions-to-keep: 2
+```
+
+<br>
+
+### Delete oldest x number of versions and keeping y latest versions of a package hosted in a different repo than the workflow
+
+To delete oldest x number of versions while keeping minimum y latest versions of a package hosted in a different repo than the workflow the __package-name__, __num-old-versions-to-delete__, __min-versions-to-keep__, __owner__, __repo__, and __token__ inputs are required.
+
+The [token][token] needs the delete packages and read packages scope. It is recommended [to store the token as a secret][secret]. In this example the [token][token] was stored as a secret named __GITHUB_PAT__.
+
+__Example__
+
+Delete the oldest 3 version and always keep atleast 2 versions of a package hosted in a different repo than the workflow
+
+```yaml
+- uses: actions/delete-package-versions@v1
+  with:
+    owner: 'github'
```

Comment:
can you add example for ignore versions as well?

Reply:
Sure will add this.